neqo-bin/Cargo.toml

```diff
@@ -34,7 +34,7 @@ neqo-crypto = { path = "./../neqo-crypto" }
 neqo-http3 = { path = "./../neqo-http3" }
 neqo-qpack = { path = "./../neqo-qpack" }
 neqo-transport = { path = "./../neqo-transport" }
-qlog = { version = "0.12", default-features = false }
+qlog = { version = "0.13", default-features = false }
```

Comment:
How about promoting `qlog` to a workspace dependency? Thus a version bump would have to touch a single file only.

https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table

Reply:
Is there a downside if not every workspace member actually needs the dependency? (Such as `fuzz`.)

Reply:
I am not aware of one @larseggert. We have been using them quite extensively in https://github.com/libp2p/rust-libp2p/blob/bee8199c0db79e2b621e3a076ed09ece65337553/Cargo.toml#L73-L127.